.all-contributorsrc

```diff
@@ -0,0 +1,105 @@
+{
+  "projectName": "hg",
+  "projectOwner": "House of Geeks",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "none",
+  "contributors": [
+    {
+      "login": "ankiiitraj",
+      "name": "Ankit Raj",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/48787278?v=4",
+      "profile": "https://github.com/ankiiitraj",
+      "contributions": [
+        "code"
+      ]
+<<<<<<< HEAD
```

Comment:
whoops . . . forgot these?

Reply:
Don't know-how, it isn't visible in my code editor, showing only on GitHub.😕
Will fix !